woocommerce/wc-admin
# Use `wc-api` for Reports.

Fixes #856.

Introduces `wc-api`/`fresh-data` for Reports in order to solve the `TypeError` issue when fetching multiple pages of report data.

### Detailed test instructions:

* Go to wp-admin > Analytics > Revenue
* Set the Date Range to Last Year
* Verify no console error (cannot read property 'data' of null)
* Visit all other report types and verify function
